getting_started/basic_walkthrough.adoc

```diff
@@ -103,27 +107,26 @@ MongoDB pod then starts up and displays its newly-assigned IP address.
 
 In this section, you will configure a GitHub webhook to automatically trigger a
 rebuild of your application whenever you push code changes to your forked
-repository.
+repository. This involves adding the Github webhook URL from your application
+into your Github repository. You obtain this webhook from these locations:
```

Comment:
@danmacpherson OK. I looked at this more closely. You changed the steps to dot points. This makes sense. I'll allow it :P